rust-runtime/aws-smithy-http/src/byte_stream/bytestream_util.rs

```diff
@@ -150,32 +171,54 @@ impl FsBuilder {
         let buffer_size = self.buffer_size;
 
         if let Some(path) = self.path {
-            let path_buf = path.to_path_buf();
-            let file_size = self.file_size.unwrap_or(
-                tokio::fs::metadata(path)
+            let offset = self.offset.unwrap_or(DEFAULT_OFFSET);
```

Comment:
can you fix this to panic if path and file are both set?

Reply:
Instead, I'll make it impossible to construct one with both.

Reply:
nm, I couldn't accomplish this in a nice way that didn't mess with the current `ByteStream` APIs for `from_path` and `from_file`